src/fheroes2/gui/ui_text.h

```diff
@@ -141,6 +141,9 @@ namespace fheroes2
         void set( const std::string & text, const FontType fontType );
         void set( std::string && text, const FontType fontType );
 
+        // This method modifies the underlying text and ends it with '...' the text if it is longer than provided width.
```

Comment:
Hi @ihhub ! Just a small correction I noticed here.
```suggestion
        // This method modifies the underlying text and ends it with '...' if it is longer than the provided width.
```